jdaviz/core/unit_conversion_utils.py

```diff
@@ -0,0 +1,398 @@
+from astropy import units as u
+import itertools
+
+from jdaviz.core.custom_units_and_equivs import (PIX2,
+                                                 SPEC_PHOTON_FLUX_DENSITY_UNITS,
+                                                 _eqv_pixar_sr,
+                                                 _eqv_flux_to_sb_pixel)
+
+__all__ = ["all_flux_unit_conversion_equivs", "check_if_unit_is_per_solid_angle",
+           "combine_flux_and_angle_units", "create_equivalent_angle_units_list",
+           "create_equivalent_flux_units_list",
+           "create_equivalent_spectral_axis_units_list",
+           "flux_conversion_general", "handle_squared_flux_unit_conversions",
+           "supported_sq_angle_units", "units_to_strings"]
+
+
+def all_flux_unit_conversion_equivs(pixar_sr=None, cube_wave=None):
+    """
+    Combines commonly used flux unit conversion equivalencies for
+    translations between flux units and between flux and surface brightness
+    units.
+
+    - Flux to flux per square pixel
+    - Flux to flux per steradian if ``pixar_sr`` is provided.
+    - Spectral density conversions (e.g. Jy to erg/s/cm2/A), if ``cube_wave``
+      is provided
+
+
+    Parameters
+    ----------
+    pixar_sr : float, optional
+        Pixel scale factor in steradians.
+    cube_wave : astropy.units.Quantity, optional
+        A reference wavelength or frequency value(s).
```

Comment:
Need blank line before Returns. It is currently not rendering properly.

```suggestion
        A reference wavelength or frequency value(s).

```

https://jdaviz--3228.org.readthedocs.build/en/3228/api/jdaviz.core.unit_conversion_utils.all_flux_unit_conversion_equivs.html#jdaviz.core.unit_conversion_utils.all_flux_unit_conversion_equivs